lib/web/scripts/install_node.go

```diff
@@ -46,3 +52,24 @@ exit 1
 var installNodeBashScript string
 
 var InstallNodeBashScript = template.Must(template.New("nodejoin").Parse(installNodeBashScript))
+
+// MarshalLabelsYAML returns a list of strings, each one containing a label key/value pair.
+// This is used to create yaml sections within the join scripts.
+func MarshalLabelsYAML(resourceMatcherLabels types.Labels) ([]string, error) {
```

Comment:
I find it surprising we don't already have a function for this. It would be good to add a few tests for `MarshalLabelsYAML`.

Reply:
I added a couple of tests